packages/hw-app-ada/src/Ada.js

```diff
@@ -32,6 +33,20 @@ const INVALID_PATH = 0x5001;
 const INVALID_PATH_LENGTH = 0x5002;
 const INDEX_NAN = 0x5003;
 
+const chunkResult = (response: Buffer, chunks) => {
```

Comment:
How does this ensure the whole data is split?

Also `response` parameters seems to be a too specific name, cannot this just be a general function?

Reply:
It does not, it chunks only what you want. Also renamed the function